pay-api/src/pay_api/models/invoice.py

```diff
@@ -131,7 +136,7 @@ class Meta(BaseSchema.Meta):  # pylint: disable=too-few-public-methods
         """Returns all the fields from the SQLAlchemy class."""
 
         model = Invoice
-        exclude = []
+        exclude = ['corp_type']
```

Comment:
exclude the relationship corp_type, we don't want it showing up in our regular invoice schema.